draft-ietf-jsonpath-base.md

```diff
@@ -208,8 +210,18 @@ Query:
 Argument:
 : Short name for the value a JSONPath expression is applied to.
 
+Location:
+: the position of a value within the argument consisting of a possibly
```

Comment:
This doesn't say what these names and indexes do.
(I would prefer "can be imagined as" over "consists of".)

Reply:
I've amplified the text a bit, but stopped short of providing a formal, recursive definition.